source/common/ssl/ssl_socket.cc

```diff
@@ -215,24 +215,32 @@ std::string SslSocket::uriSanLocalCertificate() {
   return getUriSanFromCertificate(cert);
 }
 
-std::string SslSocket::sha256PeerCertificateDigest() {
+const std::string& SslSocket::sha256PeerCertificateDigest() {
```

Comment:
sorry to ask this, but as long as you are in here, do you mind making this method const (and then making the member mutable)?

Reply:
Sure, I can do that.